www/scss/partials/_contact.scss

```diff
@@ -32,13 +37,42 @@
 .contact-content .block-title {
   color: $fsek-orange;
   font-size: 17px;
+  overflow: visible;
 }
 
 .contact-content .contact-button {
   background-color: $fsek-orange !important;
   margin-top: 20px;
```

Comment:
Add a `margin-bottom` here for material so the button isn't on the complete bottom of the screen